nix/pkgs/haskell/haskell.nix

```diff
@@ -61,6 +61,8 @@ let
           # These need R
           plutus-core.components.benchmarks.cost-model-test.buildable = lib.mkForce false;
           plutus-core.components.exes.generate-cost-model.buildable = lib.mkForce false;
+          # Contains tests
```

Comment:
This comment doesn't really explain: I think you want to say something like "This is a package that contains support for doing testing, so we're not interested in cross-compiling it"